MissionScanner.cpp

```diff
@@ -71,7 +71,7 @@ std::vector<std::string> FindMissionPaths(const std::vector<std::string>& argume
 		}
 		else
 		{
-			throw std::runtime_error("The following path was provided, but does not appear as a valid file path or directory.");
+			throw std::runtime_error("The path " + argument + " was provided, but does not appear as a valid file path or directory.");
```

Comment:
Slightly debatable suggestion, though I tend to put the fixed portion of error messages first, with the variable portion at the end. I find that makes for more easily searchable error messages.